src/components/common/HorizontalList/HorizontalList.module.scss

```
@@ -10,12 +13,20 @@
     display: flex;
     flex-direction: row;
     width: fit-content;
+    height: fit-content;
+
+    margin-bottom: 24px;
+    margin-top: 40px;
 
-    // TODO: consider moving this margin definition to component which uses this list
     > * {
       display: block;
       flex: 0 0 auto;
-      margin: 24px 12px;
+      margin-right: 16px
```

Last child shouldn't have `margin-right`, should it?

```suggestion

      margin-right: 16px;
      &:last-child {
        margin-right: 0;
      }
```